controllers/lvmcluster_controller.go

```diff
@@ -230,7 +233,7 @@ func (r *LVMClusterReconciler) updateLVMClusterStatus(ctx context.Context, insta
 		return err
 	}
 
-	expectedVgCount, err := r.getExpectedVgCount(ctx, instance)
+	expectedVGCount, err := r.getExpectedVgCount(ctx, instance)
```

Comment:
Could you rename the `getExpectedVgCount` function as well to `getExpectedVGCount`?